src/api/svg.ts

```diff
@@ -987,6 +1019,17 @@ export const drawSvg = async (
     //     thickness: 1
     //   })
     // })
+    if (elt.tagName === 'image') return parseImage(elt);
     return runners[elt.tagName]?.(elt);
   }, Promise.resolve());
-};
+}
+
+export const drawSvgSync = (
```

Comment:
I fixed the implementation, but it's a bit different from your suggestion. I didn't implement the `PDFSvg` class because it would be a braking change of the current `drawSvg` api.